subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -3905,12 +3905,12 @@ static inline u32_t isr_close_adv(void)
 		if ((_radio.state == STATE_CLOSE) &&
 		    (!_radio.advertiser.is_hdcd)) {
 			u32_t ticker_status;
-			u8_t random_delay;
+			u16_t random_delay;
 
 			entropy_nrf_get_entropy_isr(_radio.entropy,
-						    &random_delay,
+						    (void *)&random_delay,
```

Comment:
Unrelated to this PR but this hints at `entropy_nrf_get_entropy_isr()` needing fixing so that it takes directly a void pointer, instead of requiring its users to do such explicit typecasting.

Reply:
There's a refactoring going on right now for that driver (yet another one)